src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -559,7 +567,11 @@ def make_readable(folder_name):
         #     copy_directory(dotnet_src_directory, dotnet_dst_directory, verbose_output=False)
 
         input_artifacts = os.path.join(workitem_payload_directory, "collectAssembliesDirectory", coreclr_args.collection_name)
-        exclude_directory = ['Core_Root'] if coreclr_args.collection_name == "coreclr_tests" else []
+
+        exclude_directories = directories_to_ignore
+        if coreclr_args.collection_name == "coreclr_tests":
+            exclude_directories += ['Core_Root']
```

Comment:
I think this might mutate `directories_to_ignore` too. Not sure if that is intentional.

Reply:
Well, not intentional, but also not problematic. `exclude_directories = directories_to_ignore` needs to be a copy; what's the better python-ic way?

Reply:
I think `exclude_directories = list(directories_to_ignore)` would work.

Reply:
ok, I updated the code to use `list()`